optaplanner-operator/src/k8s/openshift.yml

```diff
@@ -114,7 +114,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: solverreconciler-cluster-role
+  name: optaplannersolverreconciler-cluster-role
```

Comment:
Maybe `optaplanner_solver_reconciler-cluster-role`? Or does that go against some convention? These long unstructured names are hard to read.

Reply:
I agree with your point, however, this file is generated by the `javaoperatorsdk` and then manually edited, mainly by adding permissions to various resources the operator has to work with.

If we want to fix the names, we introduce more manual interventions with a risk of inconsistencies. I will take a look if we can somehow adjust how the name is generated.

Reply:
Ok. If it becomes too complicated, let's scrap the idea indeed.

Reply:
One annotation parameter did the trick; fixed.